.github/workflows/devbuild.yml

```diff
@@ -159,7 +159,7 @@ jobs:
           VERBOSE=1 USE_CLANG_TIDY=OFF \
           BUILD_QT=OFF \
           CMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
-          CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3)"
+          CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3) -DUSE_PYTEST_HELPER_BINDING=ON"
```

Comment:
`DUSE_PYTEST_HELPER_BINDING` flag is used to build the pytest helper.

Reply:
most of the time we need this flag, since almost all builds are used for testing.